src/as/math/MathUtils.ts

```diff
@@ -12,46 +12,28 @@
  * @see <a href="https://github.com/mrdoob/three.js/blob/master/src/math/Math">src/math/Math</a>
  */
 
-export const DEG2RAD: f32 = Mathf.PI / 180
-export const RAD2DEG: f32 = 180 / Mathf.PI
+const DEG2RAD: f32 = Mathf.PI / 180
+const RAD2DEG: f32 = 180 / Mathf.PI
 
-export function toString(source: i32, radix: i8): string {
-	let result: string = ''
-
-	return result
-}
-
-function _toHexString(integer: i32): string {
-	//number mod 16
-	//lookup mod in hex strings
-	//add to the beginning of the string
-	//divide by 16
-	//repeat while number > 0
-
-	var hexValues: string[] = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'A', 'B', 'C', 'D', 'E', 'F']
-	var mod: i32 = 0
-	var result: string = ''
-
-	while (integer > 1) {
-		mod = integer % 16
-		result = hexValues[mod] + result
-		integer = integer / 16
+/**
+ * Returns the respective basic value converted to a string.
+ */
+export function toString(number: i32, radix?: i8): string {
+	if (radix === null || radix > 1 && radix < 17) {
+		return number.toString(radix)
```

Comment:
```suggestion
export function toString(number: i32, radix: i32 = 10): string {
   return number.toString(radix);
}
```

Internally I32#toString already has checks so all this checks unnecessary

Reply:
- OK but what happens for values not between 2 and 36?
- changing type from i8 to i32 is not relevant for values of that order, you might be using a bigger register.
- default value to 10 is more explicite. OK.
- comma is not part of the project linting rules, please!

Reply:
> OK but what happens for values not between 2 and 36?

Will throw trap: https://github.com/AssemblyScript/assemblyscript/blob/master/std/assembly/util/number.ts#L373

Reply:
Good! Will you remove the comma?

Reply:
> Will you remove the comma?

Just rewrite as you want. It not necessary apply suggestions as is